### ISSUE TITLE
Support GHC-8.8.3

### ISSUE THREAD

**Issue by chshersh** [utdemir/ghc-musl#3]
Thanks for writing this package!

I see that the latest uploaded Docker container supports GHC version is 8.8.1. It would be lovely to support 8.8.3 :slightly_smiling_face:

**Comment by utdemir** [comment]
Thank you. I'll update it, and also package GHC 8.10 (if it has landed in nixpkgs). It will take a few days since I have to leave my computer overnight each time I compile a new GHC :).

**Comment by utdemir** [comment]
This issue is blocked on: https://github.com/NixOS/nixpkgs/issues/83926

**Comment by utdemir** [comment]
Now blocked on another issue: https://github.com/NixOS/nixpkgs/issues/84670

**Comment by utdemir** [comment]
I recently uploaded `v7` series of compilers with `ghc8101`, `ghc883` and `ghc865`.
